modules/draco/src/draco-loader.js

```diff
@@ -38,7 +38,7 @@ async function parse(arrayBuffer, options, context, loader) {
   try {
     // TODO passing in options causes CI failures...
     // @ts-ignore
-    return dracoParser.parseSync(arrayBuffer);
+    return dracoParser.parseSync(arrayBuffer, {extraAttributes: options.extraAttributes || null});
```

Comment:
To enable options for multiple loaders to be specified in one object, we collect the options for each load in an options subobject (`options.draco.extraAttributes`).

Also, there is code that warns for unrecognized options by matching keys against default values:  https://github.com/visgl/loaders.gl/pull/940/files#diff-913f488b37da14e2ce19af746d5500deR21

Reply:
Got it! I'll change the option to nest in `draco`.

Reply:
Code is now updated to nest the option in the `draco` namespace. Let me know if this works!